MAUI/NumericEntry/UpDown-Button.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Use UpDown Button in .NET MAUI NumericEntry | Syncfusion
+description: Learn here all about how to use UpDown Button (SpinButton) in Syncfusion .NET MAUI NumericEntry (SfNumericEntry) control and more.
+platform: MAUI
+control:  SfNumericEntry
+documentation: ug
+---
+
+# UpDown Button in .NET MAUI NumericEntry
+
+This section describes how to change value in the `NumericEntry` control using keys, mouse scrolling and the up-down button.
```

Comment:
This section describes how to change the value in the `NumericEntry` control using keys, mouse scrolling, and the up-down button.

Reply:
The requested content modification has been successfully addressed